server/publications/collections/products.js

```diff
@@ -416,14 +412,16 @@ Meteor.publish("Products", function (productScrollLimit = 24, productFilters, so
   const productIds = Products.find(selector, {
     sort,
     limit: productScrollLimit
+  }, {
+    fields: {
+      _id: 1
+    }
   }).map((product) => product._id);
 
-  let newSelector = { ...selector };
+  let newSelector = _.omit(...selector, ["hashtags", "ancestors"]);
```

Comment:
both branches of the conditional below perform the `omit` operation, so I moved it here and combined it with the initial spread/clone